lib/modules/play/widgets/typing_indicator.dart

```diff
@@ -2,25 +2,35 @@ import 'package:flutter/material.dart';
 import 'package:simple_animations/simple_animations.dart';
 
 class TypingIndicator extends StatelessWidget {
-  const TypingIndicator({Key? key}) : super(key: key);
+  const TypingIndicator({
+    this.onTap,
+    Key? key,
+  }) : super(key: key);
+
+  final VoidCallback? onTap;
 
   @override
   Widget build(BuildContext context) {
-    return Center(
-      child: MirrorAnimation<double>(
-        tween: Tween(begin: 0, end: 1),
-        curve: Curves.easeInOut,
-        duration: const Duration(milliseconds: 900),
-        builder: (context, child, tween) {
-          var text = '';
-          for (var i = 1; i < 4; i++) {
-            if (tween > .3 * i) text += ' •';
-          }
-          return Padding(
-            padding: const EdgeInsets.all(16),
-            child: Text(text),
-          );
-        },
+    const l = 5;
```

Comment:
`l` is not that much more descriptive than just `5`. It's also very easy to misread as `1`. Please rename it